### PR TITLE
CLC-5773, script delegates to OEC rake tasks

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CLC-5773

This script can run any of the OEC rake tasks, given the proper command line args. See usage info.

### CODE REVIEW

File: script/run-oec-task.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  # Enable rvm and use the correct Ruby version and gem set.
+  [[ -s "${HOME}/.rvm/scripts/rvm" ]] && . "${HOME}/.rvm/scripts/rvm"
+  source .rvmrc
+
+  export RAILS_ENV=production
+  export LOGGER_STDOUT=only
+  export LOGGER_LEVEL=INFO
+  echo "$(date): Run oec:${REQUESTED_TASK} task on $(hostname -s)" | ${LOGIT}
+
+  echo 'cd deploy'
```
Comment (paulkerschen) [inline script/run-oec-task.sh]: @johncrossman Is this supposed to be inside an `echo`?

**Comment by paulkerschen** [comment]
@johncrossman Do we need any of the JRUBY_OPTS that the other scripts set? I would not say I'm well informed about what all those options do.

File: script/run-oec-task.sh (hunk 1 above)
Comment (johncrossman) [inline script/run-oec-task.sh]: Good catch, @paulkerschen  A remnant of my debugging

**Comment by johncrossman** [comment]
@paulkerschen I would expect that env var to be picked up by `source .bash_profile` but I'll throw it in. Seems silly to have that copy 'n pasted across all scripts. :)

**Comment by paulkerschen** [comment]
@johncrossman Good point, it should come in with the profile. I guess we should check in our ample spare time whether all this explicit option-setting is needed.

I like the flexibility of this wrapper. Superstitiously waiting on T though I wouldn't expect T to touch this one.

**Pull request merged**